app/controllers/owner/repositories.js

```diff
@@ -1,6 +1,30 @@
 import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { and, equal, or } from '@ember/object/computed';
+import { DEFAULT_INSIGHTS_INTERVAL } from 'travis/services/insights';
 
 export default Controller.extend({
+  queryParams: ['tab', 'timeInterval'],
+
+  features: service(),
+
   isLoading: false,
   page: 1,
+  tab: null,
+  hasNoBuilds: false,
+  requestPrivateInsights: true,
+
+  timeInterval: null,
+  defaultTimeInterval: DEFAULT_INSIGHTS_INTERVAL,
+  currentTimeInterval: or('timeInterval', 'defaultTimeInterval'),
```

Comment:
It could, but then `timeInterval` shows up in the URL even if you're on the repositories tab. Which is a little weird, although it may help resolve an issue Maria noticed where the tab doesn't highlight correctly when `timeInterval` is not in the URL

Reply:
You could set the query parameter value to default one when switching to repositories on the route's [`resetController` hook](https://api.emberjs.com/ember/3.8/classes/Route/methods/resetController?anchor=resetController). E.g.
```js
resetController(controller) {
  this._super(...arguments);
  if (this.controller.isInsights) {
    this.controller.set('timeInterval', DEFAULT_INSIGHTS_INTERVAL);
  }
}
```
This should remove the query parameter from URL when switching from Insights

Reply:
Neat! I didn't think of that and I really like that solution 😄

Reply:
I ran into some issues while trying to implement this solution, when I thought of another one. Since the queryParam doesn't show up in the URL when it's equal to the default value, Updating the Repositories `link-to` to `(query-params tab=null timeInterval=defaultTimeInterval)` resolves the issue.

Reply:
That's a great solution, even better than mine :) But would it make sense to to also set a meaningful name for repositories tab, like `tab=repositories`. I know, it will never be visible for users (if not filled by hand), but still :)

Reply:
Yeah I really like that, I'll add it in 😄

Reply:
Change made :)